### PR TITLE
Document signature verifications

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-2077

### CODE REVIEW

File: docs/modules/ROOT/pages/signing.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+== Sigstore Levels
+
+There are different levels of Sigstore adoption. These can be done
+https://www.linuxfoundation.org/blog/blog/adopting-sigstore-incrementally[incrementally]. The
+Enterprise Contract can be used to validate artifacts regardless of the level.
+
+=== Long-Lived Keys
+
+Using long-lived keys is the easiest and simplest way to adopt Sigstore. For verification, there are
+two requirements is the public signing key. First, the public key itself, and second a flag to
```
Comment (joejstuart) [inline docs/modules/ROOT/pages/signing.adoc:16]: typo maybe?

File: docs/modules/ROOT/pages/signing.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+[,bash]
+----
+ec validate image --public-key=cosign.pub --ignore-rekor --image $IMAGE
+----
+
+=== Long-Lived Keys with Auditability
+
+This approach uses https://docs.sigstore.dev/rekor/overview/[Rekor] to track when the signing key is
+used. When using the production public instance of Rekor, the only requirement is the public key:
```
Comment (joejstuart) [inline docs/modules/ROOT/pages/signing.adoc:27]: "production public" or maybe just "public"?

**Review by joejstuart — APPROVED** [review]

File: docs/modules/ROOT/pages/signing.adoc (hunk 1 above)
Comment (lcarva) [reply docs/modules/ROOT/pages/signing.adoc:16]: Which part? :sweat_smile:

Comment (lcarva) [reply docs/modules/ROOT/pages/signing.adoc:16]: I see it now... :facepalm:

**Pull request merged**